gen/gen.go

```diff
@@ -119,23 +140,67 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 	return ioutil.WriteFile(filepath.Join(dirProblem, "cases_test.go"), src, 0666)
 }
 
-func getPath(jFile string) (jPath, jOrigin, jCommit string) {
+func getLocal(jFile string) (jPath, jOrigin, jCommit string) {
 	// Ideally draw from a .json which is pulled from the official x-common
 	// repository.  For development however, accept a file in current directory
 	// if there is no .json in source control.  Also allow an override in any
 	// case by environment variable.
-	if jPath = os.Getenv("EXTEST"); jPath > "" {
+	if jPath := os.Getenv("EXTEST"); jPath > "" {
 		return jPath, "local file", "" // override
 	}
+
 	c := exec.Command("git", "log", "-1", "--oneline", jFile)
 	c.Dir = dirMetadata
-	ori, err := c.Output()
+	origin, err := c.Output()
 	if err != nil {
 		return "", "local file", "" // no source control
 	}
 	if _, err = os.Stat(filepath.Join(c.Dir, jFile)); err != nil {
 		return "", "local file", "" // not in source control
 	}
+
 	// good.  return source control dir and commit.
-	return c.Dir, "exercism/x-common", string(bytes.TrimSpace(ori))
+	fmt.Printf("Fetching canonical data from a local source.\nHave you ensured it is up to date?\n")
```

Comment:
I don't know about you, but I forget to do this all the time! A little reminder here won't go amiss. :smile: I also thought it would be good to indicate if the generator is collecting locally or remotely.